src/pybind/mgr/telemetry/module.py

```diff
@@ -327,6 +331,110 @@ def gather_crashinfo(self) -> List[Dict[str, str]]:
             crashlist.append(c)
         return crashlist
 
+    def gather_perf_counters(self) -> Dict[str, dict]:
+        # Initialize 'perf schema' dict
+        perf_schema: Dict[str, dict] = defaultdict(dict)
+
+        # Get initial info, condense among like daemons (i.e. 'osd' = 'osd.0'
+        # + 'osd.1' + 'osd.2') and put results into 'perf schema'
+        initial_info = self.get_all_perf_counters()
+        for daemon in initial_info:
+            daemon_type = daemon[0:3] # i.e. 'mds', 'osd', 'rgw'
+            if daemon_type not in perf_schema:
+                perf_schema[daemon_type] = initial_info[daemon]
+            else:
+                for collection in perf_schema[daemon_type]:
+                    for field in perf_schema[daemon_type][collection]:
+                        if type(perf_schema[daemon_type][collection][field]) != str:
+                            perf_schema[daemon_type][collection][field] += \
+                                    initial_info[daemon][collection][field]
```

Comment:
Ah, I didn't think of the `throttle-msgr_dispatch_throttler-hbserver` format since that particular counter doesn't show up in my vstart cluster. Glad this was caught.